.dotnet/src/Custom/Assistants/Internal/InternalAssistantMessageClient.cs

```diff
@@ -18,16 +18,10 @@ namespace OpenAI.Assistants;
 [CodeGenSuppress("DeleteMessage", typeof(string), typeof(string))]
 internal partial class InternalAssistantMessageClient
 {
-    // CUSTOM: Remove virtual keyword.
-    /// <summary>
-    /// The HTTP pipeline for sending and receiving REST requests and responses.
-    /// </summary>
-    public ClientPipeline Pipeline => _pipeline;
```

Comment:
These customizations are no longer needed as MGC generates the property as non-virtual.